### PR TITLE
WOR-1458 - Pact Test Fixes

### PR DESCRIPTION
**Pull request by blakery**
https://broadworkbench.atlassian.net/browse/WOR-1458

* Re-enable can-i-deploy for consumer contract tests (now that https://broadworkbench.atlassian.net/browse/WOR-890 is finished)
* publish all pacts in output directory to the broker

### CODE REVIEW

File: .github/workflows/consumer_contract_tests.yml [hunk 1]
```diff
@@ -121,50 +122,72 @@ jobs:
...
+        with:
+          name: ${{ env.PACTS_ARTIFACT }}
+          path: ${{ env.PACTS_OUTPUT_DIR }}
+          retention-days: 1
 
   publish-contracts:
     runs-on: ubuntu-latest
     needs: [ bump-check, init-github-context, bpm-consumer-contract-tests, regulated-tag-job ]
+    strategy:
+      matrix:
```
Comment (cahrens) [inline .github/workflows/consumer_contract_tests.yml:142]: Why does this need matrix? Matrix has caused problems in the past with incorrect github actions runs getting linked, and I don't see that we are iterating over anything.

Comment (cahrens) [reply .github/workflows/consumer_contract_tests.yml:142]: Unless `pact-path` is more than a single path?

**Comment by cahrens** [comment]
Can you provide a link to the pact broker showing where these got published? I'm just looking to understand more about how we know this is actually working.

File: .github/workflows/consumer_contract_tests.yml (hunk 1 above)
Comment (cahrens) [reply .github/workflows/consumer_contract_tests.yml:142]: Yes, looking at the actual ticket for this change I see it is about publishing multiple pacts. So I understand now the matrix call.

**Review by cahrens — APPROVED** [review]
I would love for you to walk through the pact broker with us again.

**Review by marctalbott — APPROVED** [review]

**Pull request merged**